btf/traversal.go

```diff
@@ -139,3 +162,15 @@ func walkType(typ Type, fn func(*Type)) {
 		panic(fmt.Sprintf("don't know how to walk Type %T", v))
 	}
 }
+
+func walkMembers(members []Member, mode walkMode, fn func(*Type)) {
```

Comment:
Would have been nice to somehow use this for slices of `Member`, `FuncParam` and `Var` but I see that making that work wouldn't really be worth the effort.

Reply:
We could make it work by having all those three implement an interface, and then use that as a constraint to a generic `walkSlice[T walkable]([]T, mode walkMode, func(*Type))`. The downside is that the generic function wouldn't be inlined probably, and that `walkSlice[Member]` uses dynamic instead of static dispatch. See https://planetscale.com/blog/generics-can-make-your-go-code-slower for an explanation. Feel free to give it a try if you want to, I think it'll probably end up being a wash in terms of speed / lines of code.